packages/browser/src/tracing/browserTracingIntegration.ts

```diff
@@ -149,7 +149,22 @@ export interface BrowserTracingOptions {
    *
    * Default: []
    */
-  ignoreResourceSpans: Array<string>;
+  ignoreResourceSpans: Array<'resouce.script' | 'resource.css' | 'resource.img' | 'resource.other' | string>;
+
+  /**
+   * Spans created from
+   * [`performance.mark(...)`](https://developer.mozilla.org/en-US/docs/Web/API/Performance/mark)
+   * and
+   * [`performance.measure(...)`](https://developer.mozilla.org/en-US/docs/Web/API/Performance/measure)
+   * calls will not be emitted if their names match strings in this array.
+   *
+   * This is useful, if you come across `mark` or `measure` spans in your Sentry traces
+   * that you want to ignore. For example, sometimes, browser extensions or libraries
+   * emit these entries on their own, which might not be relevant to your application.
+   *
+   * Default: [] - By default, all `mark` and `measure` entries are sent as spans.
+   */
+  ignoreMeasureSpans: Array<string | RegExp>;
```

Comment:
As this not only applies to the `measure` spans...should we call this `ignorePerformanceAPISpans` or `ignoreWebPerformanceSpans`? 🤔

No hard opinion on that though as my proposed name might be too wide in its description.

Reply:
I had exactly the same feeling. Let's make a decision fast, as I agree the name doesn't really include mark spans.

Reply:
And this is how we could extend this if people need granular options:

```js
ignorePerformanceAPISpans: {
  mark:
  measure:
  ...
}
```

Reply:
[ae2cfb0](https://github.com/getsentry/sentry-javascript/pull/16443/commits/ae2cfb0c049cefb2c83abf8b36b6dcccf42dbdff)